deliverables/README.md

```diff
@@ -21,48 +21,110 @@ For projects on Code Academy, or other external exercises, please just paste a l
+
 What are tags?  
+Are keywords surronded by ankle braclets. They come in pairs, a start and end one.
+
 What are attributes?  
+Act as a modifier to the HTML element. They to consist of different tags. 
+
 What are forms?  
+A form allows one to enter data to a web page. They collect the users input. 
+
 What is a div?  
+It groups elements together. The most common method for identyfying structure. 
 
 What are selectors?  
-What are properties?  
+They select specific elements that are unique on the page.
+
+What are properties?
+Properties effect how the code is represeneted.
+
 What are values?  
+Values are specific to the property. For example, red would be a value to the property color.
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You use ID's to choose the style. If you don't choose the style, the web program should have a default setting unique to the program.
 
+What are two CSS attributes you can change to push an element around on the page? 
+Float and margin can move an element around on the page.
+
+What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use to change font color, change font size and change the type of font.
 ---
 ### Section 3: Git and Github  
 What is Git?  
+A distributed control system. Basically a control system for software development. 
+
 What is SCM?  
+Scheme Implementation.
+
 What is a VCS?  
+Veritas Cluster Server
+
 Why is Git useful for a developer?  
+Git is useful for a developer because they can share work, and stem off of other peoples work. It is also a good way to build of your own work.
+
 Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
+A group can work on colabortive projects together.
+
+How do you create a new Git repository for a project locally? 
+You clone the project, and create a new project with the download data.
+
 How do you create it on Github?  
+In the upper-right corner of any page, click , and then click New repository.
+Create a short, memorable name for your repository. ...
+Optionally, add a description of your repository. ...
+Choose between creating a public or private repository. ...
+Select Initialize this repository with a README.
+
 How do you commit changes?  
+You must fork it and then go to readme tab and then click on the pen in right corner to edit.
+
 What is the difference between staging and committing changes?  
+One adds a command line and the other adds a new remote.
+
 What is the difference between committing your changes and pushing them to Github?  
+Pushing them back to Github saves it as the new file in your repo, when commmit simply records a small snapshot.
```

Comment:
Close! Committing saves the changes locally - that is, on your machine - and pushing to Github saves your changes in your repository online on Github's servers.